cmd/hz/thrift/ast.go

```diff
@@ -55,21 +59,26 @@ func astToService(ast *parser.Thrift, resolver *Resolver) ([]*generator.Service,
 		service := &generator.Service{
 			Name: s.GetName(),
 		}
+		baseDomain := ""
+		domainAnno := getAnnotation(s.Annotations, ApiBaseDomain)
+		if len(domainAnno) == 1 {
+			baseDomain = domainAnno[0]
+			if args.CmdType == meta.CmdClient {
```

Comment:
If only `args.CmdType` is used in this function, it is better to take  `args.CmdType` as a parameter rather than `*config.Argument`?
`*config.Argument` contains too many infos.

Reply:
I felt that it might use other parameters later, so I passed the whole `config.Argument`.

Reply:
If it may be used in the future, you also need to check which fields are used. If you really need the entire parameter, you can also modify it at that time (private function)

Reply:
done